cdap-ui/app/features/workflows/templates/detail.html

```diff
@@ -1,39 +1,199 @@
+            <span class="icon-workflow"></span>
+            <small> Type: Workflow </small>
+          </div>
+          <p> Description goes here. </p>
+          <div class="program-tags">
+            <a class="btn">Dataset</a>
+            <a class="btn">Apps</a>
+            <a class="btn">
+              <span class="fa fa-plus"></span>
+            </a>
+          </div>
+        </div>
+        <div class="col-sm-8 col-md-6">
+          <div class="button-bar text-right">
+            <my-start-stop-button
+              type="workflows"
+              is-stoppable="false"
+              is-restartable="true">
+            </my-start-stop-button>
+            <div class="btn-group dropdown" dropdown>
+              <div class="btn btn-lg btn-default dropdown-toggle" dropdown-toggle>
+                <span class="fa fa-ellipsis-h"></span>
+                <span> More </span>
+              </div>
+              <ul class="dropdown-menu" role="menu">
+                <li>
+                  <div ng-click="RunsController.openHistory()"
+                     class="btn btn-lg btn-default">
+                    <span class="fa fa-clock-o"></span>
+                    <span> History </span>
+                  </div>
+                </li>
+                <li>
+                  <div class="btn btn-lg btn-default" ng-click="RunsController.openSchedules()">
+                    <span class="fa fa-database"></span>
+                    <span> Schedules </span>
+                  </div>
+                </li>
+                <li ui-sref="admin.namespace.detail.apps.metadata({nsadmin: $state.params.namespace, appId: $state.params.appId})">
+                  <div class="btn btn-lg btn-default">
+                    <span class="fa fa-cogs"></span>
+                    <span> Manage </span>
+                  </div>
+                </li>
+              </ul>
+            </div>
+          </div>
+        </div>
+      </div>
     </div>
   </div>
-  <div class="col-sm-6 text-right">
-    <my-start-stop-button
-      type="workflows"
-      is-stoppable="false"
-      is-restartable="true">
-    </my-start-stop-button>
-  </div>
-</div>
+  <div class="run-level">
+    <div class="container">
+      <div class="run-level-inner">
+
+        <div dropdown is-open="dropdown.isopen" class="flex-item">
+          <a class="btn btn-default dropdown-toggle"
+             dropdown-toggle ng-disabled="RunsController.runs.length === 0">
+            <span class="fa fa-circle" ng-class="{'text-primary': RunsController.runs.selected.status === 'COMPLETED', 'text-success': RunsController.runs.selected.status === 'STARTED' || RunsController.runs.selected.status === 'RUNNING', 'text-danger': RunsController.runs.selected.status === 'FAILED' || RunsController.runs.selected.status === 'KILLED', 'text-muted': RunsController.runs.selected.status === 'STOPPED', 'text-warning': RunsController.runs.selected.status === 'SUSPENDED'}"></span>
```

Comment:
I agree we need to centralize this and use it everywhere. Let's not replicate this everywhere.